apps/win32/metro.config.js

```diff
@@ -9,42 +9,46 @@ const blacklist = require('metro-config/src/defaults/blacklist');
 const { getWatchFolders } = require('@uifabricshared/build-native');
 const { getDefaultConfig } = require('metro-config');
 
-
 module.exports = (async () => {
-    const {
-        resolver: {
-            sourceExts,
-            assetExts
-        }
-    } = await getDefaultConfig();
-    return {
-        watchFolders: getWatchFolders(),
-        resolver: {
-          assetExts: assetExts.filter(ext => ext !== 'svg'),
-          sourceExts: [
-              ...sourceExts,
-              'svg'
-          ],
-          resolveRequest: require('@office-iss/react-native-win32/metro-react-native-platform').reactNativePlatformResolver(
-              {win32: '@office-iss/react-native-win32'}
-          ),
-          blacklistRE: blacklist([
-              // This stops "react-native run-windows" from causing the metro server to crash if its already running
-              new RegExp(`${path.resolve(__dirname, 'windows').replace(/[/\\]/g, '/')}.*`),
-          ])
+  const {
+    resolver: { sourceExts, assetExts },
+  } = await getDefaultConfig();
+  return {
+    watchFolders: getWatchFolders(),
+    resolver: {
+      assetExts: assetExts.filter((ext) => ext !== 'svg'),
+      sourceExts: [...sourceExts, 'svg'],
+      blacklistRE: blacklist([
+        // This stops "react-native run-windows" from causing the metro server to crash if its already running
+        new RegExp(`${path.resolve(__dirname, 'windows').replace(/[/\\]/g, '/')}.*`),
+      ]),
+    },
+    // Metro doesn't currently handle assets coming from hoisted packages within a monorepo.  This is the current workaround people use
```

Comment:
Just making sure, we don't know what release (if any) is going to "fix" this issue, correct?

Reply:
There is no current fix in progress for this.  Basically metro/the assets get generated using a url that contains ../.., which isn't really valid as part of a url.

There was some discussion to change the url to have the path to asset as a param on the url instead of part of the url itself.  But I didn't see any progress on that happening.

Reply:
Interesting. Is the location we need consistent with regards to the RN path? Because then we could transform the URLs from relative to absolute paths where we dynamically look up the RN-win32 path. Not blocking, just a thought of whether that would be more robust.

Reply:
The URL to metro has to be relative, since its in the form:
`http://localhost:8081/asset/<url>`

The discussion in the community was that it should be changed to:
`http://localhost:8081/asset?url=<url>`
Then the ..'s would still be a valid url.